great_expectations/core/expectation_diagnostics/expectation_diagnostics.py

```diff
@@ -378,10 +395,73 @@ def _check_renderer_methods(
     @staticmethod
     def _check_linting(expectation_instance) -> ExpectationDiagnosticCheckMessage:
         """Check if linting checks pass for Expectation"""
-        # TODO: Perform linting checks instead of just giving thumbs down
+        sub_messages = []
+        passed = False
+        black_ok = False
+        isort_ok = False
+        file_and_class_names_ok = False
+
+        expectation_camel_name = rx_expectation_instance_repr.match(
+            repr(expectation_instance)
+        ).group(1)
```

Comment:
Will this ever fail? What if group doesn't have 1?

Nitpick - could we split up this logic into separate lines? Perhaps the match on one and the group on the other?